doc/Command_Index_User.tex

```diff
@@ -2093,11 +2099,11 @@ \subsubsection{Vertical spacing}
 \end{gdimension}
 
 \begin{gdimension}{lineskiplimit}
-The effective \verb=\lineskiplimit= inside of a score. If two consecutive staves have baselines \texttt{baselineskip} apart and the distance between the first staff's bottomline and the second staff's topline is less than this, then \texttt{lineskip} is added between them.
+The effective \verb=\lineskiplimit= inside of a score. If two consecutive staves have baselines \texttt{baselineskip} apart and the distance between the first staff's bottome and the second staff's top is less than this, then \texttt{lineskip} is added between them.
```

Comment:
s/bottome/bottom/

Reply:
Done

(I realized that "topline" means something different than how I was using it, so I changed "topline" and "bottomline" to "top" and "bottom".)